src/main/java/com/zuniorteam/lotto/core/LottoSeller.java

```diff
@@ -17,24 +19,34 @@ public LottoSeller(LottoMachine lottoMachine) {
         this.lottoMachine = lottoMachine;
     }
 
-    public List<Lotto> sell(int money) {
-        validate(money);
+    public List<Lotto> sell(Money money, List<Lotto> appointLottos) {
+        validate(money , appointLottos);
 
-        final int numberOfLottos = money / LOTTO_PRICE;
+        final long numberOfLottos = getNumberOfAutoLottos(money, appointLottos.size());
```

Comment:
이건 자동 로또 갯수니까 변수명을 바꾸는게 좋을 것 같아요
```suggestion
        final long numberOfAutoLottos = getNumberOfAutoLottos(money, appointLottos.size());
```

Reply:
헉,,그러네요, 감사합니다!!